File: leetcode/976. 三角形的最大周长.cc
class Solution {
public:
    int largestPerimeter(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        for (int i = 1; i <= nums.size() - 2; ++i) {
            if (nums[nums.size() - i] < nums[nums.size() - i - 1] + nums[nums.size() - i - 2]) {
                return nums[nums.size() - i] + nums[nums.size() - i - 1] + nums[nums.size() - i - 2];
            }
        }
        return 0;
    }
};

File: leetcode/976. 三角形的最大周长.java
class Solution {
    public int largestPerimeter(int[] nums) {
        Arrays.sort(nums);
        for (int i = 1; i <= nums.length - 2; i++) {
            if (nums[nums.length - i] < nums[nums.length - i - 1] + nums[nums.length - i - 2]) {
                return nums[nums.length - i] + nums[nums.length - i - 1] + nums[nums.length - i - 2];
            }
        }
        return 0;
    }
}